lib/pdf_fill/forms/va21p527ez.rb

```diff
@@ -1393,6 +1393,7 @@ def expand_prior_marital_history
             reason_for_separation = marriage['reasonForSeparation'].to_s.downcase
             marriage.merge({ 'dateOfMarriage' => split_date(marriage['dateOfMarriage']),
                              'reasonForSeparation' => reason_for_separation_lookup[reason_for_separation],
+                             'otherExplanation' => marriage['separationOtherExplanation'] || '',
```

Comment:
This change shouldn't be needed, as `otherExplanation` since we just display this value as is. We only add fields into these `expand_` prefixed functions if additional manipulation of the data is needed before being mapped.